### PR TITLE
AVX128: More instructions

### PR DESCRIPTION
**Pull request by Sonicadvance1**
What we have here is primarily, inserts, extracts, conversions, multiplies, a subtract, and a horizontal subtract.

### CODE REVIEW

**Comment by alyssarosenzweig** [comment]
I did quite a bit of surgery on this one, diff to review https://rosenzweig.io/avx13.diff

**Review by alyssarosenzweig — APPROVED** [review]
(with my changes)

File: FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp [hunk 1]
```diff
@@ -1350,4 +1350,254 @@ void OpDispatchBuilder::AVX128_VPSLLDQ(OpcodeArgs) {
   AVX128_ShiftDoubleImm(Op, ShiftDirection::LEFT);
 }
 
+void OpDispatchBuilder::AVX128_VINSERT(OpcodeArgs) {
+  const auto DstSize = GetDstSize(Op);
+  const auto Is128Bit = DstSize == Core::CPUState::XMM_SSE_REG_SIZE;
+  const auto Selector = Op->Src[2].Literal();
```
Comment (Sonicadvance1) [inline FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp:1356]: This had a bitmask dropped which needs to be ketpt.

File: FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp [hunk 2]
```diff
@@ -1350,4 +1350,254 @@ void OpDispatchBuilder::AVX128_VPSLLDQ(OpcodeArgs) {
...
+      Result.High = Convert(Size, Src.High, IROps::OP_VSXTL);
+    }
+  }
+
+  AVX128_StoreResult_WithOpSize(Op, Op->Dest, Result);
+}
+
+void OpDispatchBuilder::AVX128_VEXTRACT128(OpcodeArgs) {
+  const auto DstIsXMM = Op->Dest.IsGPR();
+  const auto Selector = Op->Src[1].Literal();
```
Comment (Sonicadvance1) [inline FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp:1583]: This had a bitmask dropped that needs to be kept.

**Review by Sonicadvance1 — COMMENTED** [review]
Other than the two comments I'm cool with the changes

File: FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp (hunk 1 above)
Comment (alyssarosenzweig) [reply FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp]: 4 eyes are better than 2.. thanks!

**Comment by alyssarosenzweig** [comment]
> Other than the two comments I'm cool with the changes

Merge at your leisure then :smile:

**Pull request merged**